analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -122,6 +122,22 @@ class MavenSupport(private val workspaceReader: WorkspaceReader) {
             }
         }
 
+        fun parseOrganizations(mavenProject: MavenProject) =
```

Comment:
As this not only parses organization tags, we probably should name this function *for* what information is parsed, and use `parseAuthors` like we already do for `Npm` and `Pub`.